lexos/static/js/scripts_upload.js

```diff
@@ -1,197 +1,197 @@
 $(function () {
-  if ($('.fa-folder-open-o')[0].style.display != 'none' &&
-    $('.fa-folder-open-o')[0].innerText.indexOf('Response') != -1) {
-    alert('Steven\'s special bug detected! Click OK and the page will load.')
-    $('.fa-folder-open-o')[0].style.display = 'none'
-  }
-
+  /*message for when you hover on the open folder icon on the top right*/
   $('[data-toggle="tooltip"]').tooltip()
-
   $('#uploadbrowse').click(function () {
     $('#fileselect').click()
   })
 
-  // ------------------- FILEDRAG -----------------------------
-
-  var allowedFileTypes = ['txt', 'xml', 'html', 'sgml', 'lexos']
-
+  //alias function
   function $id(id) {
```

Comment:
I don't think you need this. `$(document.getElementById("id"))` is equivalent to `$('#id')`